codespell_lib/data/dictionary.txt

```diff
@@ -4909,6 +4922,7 @@ candadate->candidate
 candadates->candidates
 candiate->candidate
 candiates->candidates
+candidats->candidates
```

Comment:
Also this one (although it's a valid French word... but that might explain the occurrences of it)
```suggestion
candidat->candidate
candiate->candidate
candiates->candidates
candidats->candidates
```